perf/run-benchmarks

```diff
@@ -439,6 +440,18 @@ def main() -> None:
     parser.add_argument(
         "--std-only", help="only run the default semgrep", action="store_true"
     )
+    parser.add_argument(
+        "--filter-corpus",
+        metavar="REGEXP",
```

Comment:
I would include an example of a regex string you could use (like you gave in the commit) in the help message

Reply:
Perhaps a `list` subcommand (or `--list` option) would be useful. It would list the name of the corpuses and variants in alphabetical order, so we know what to filter for without inspecting the source code.